table-binary-unicode-properties.html

```diff
@@ -12,22 +12,18 @@
       <td><a href="https://unicode.org/reports/tr18/#General_Category_Property">`ASCII`</a></td>
     </tr>
     <tr>
-      <td>
-        <ul>
-          <li>`ASCII_Hex_Digit`</li>
-          <li>`AHex`</li>
-        </ul>
-      </td>
-      <td><a href="https://unicode.org/reports/tr44/#ASCII_Hex_Digit">`ASCII_Hex_Digit`</a></td>
+      <td>`ASCII_Hex_Digit`</td>
+      <td rowspan=2><a href="https://unicode.org/reports/tr44/#ASCII_Hex_Digit">`ASCII_Hex_Digit`</a></td>
```

Comment:
i have a mild preference for always quoting all html property values

Reply:
Done.